specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3029 @@
+        {
+          "$ref": "#/definitions/JobState"
+        },
+        {
+          "$ref": "#/definitions/TasksState"
+        },
+        {
+          "$ref": "#/definitions/AnalyzeTextJobStatistics"
+        }
+      ]
+    },
+    "Pagination": {
+      "properties": {
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "JobMetadata": {
+      "properties": {
+        "displayName": {
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "jobId": {
+          "format": "uuid",
+          "type": "string"
+        },
+        "lastUpdateDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "status": {
+          "enum": [
+            "notStarted",
+            "running",
+            "succeeded",
+            "partiallySucceeded",
+            "failed",
+            "cancelled",
+            "cancelling"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": false,
+            "name": "State"
+          }
+        }
+      },
+      "required": [
+        "jobId",
+        "lastUpdateDateTime",
+        "createdDateTime",
+        "status"
+      ],
+      "type": "object"
+    },
+    "JobState": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobMetadata"
+        },
+        {
+          "$ref": "#/definitions/JobErrors"
+        },
+        {
+          "$ref": "#/definitions/Pagination"
+        }
+      ]
+    },
+    "JobErrors": {
+      "properties": {
+        "errors": {
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "AnalyzeTextJobStatistics": {
+      "properties": {
+        "statistics": {
+          "$ref": "#/definitions/RequestStatistics"
+        }
+      },
+      "type": "object"
+    },
+    "TasksState": {
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
+              "type": "integer"
+            },
+            "failed": {
+              "type": "integer"
+            },
+            "inProgress": {
+              "type": "integer"
+            },
+            "total": {
+              "type": "integer"
+            },
+            "items": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AnalyzeTextLROResult"
+              }
+            }
+          },
+          "required": [
+            "total",
+            "completed",
+            "failed",
+            "inProgress"
+          ],
+          "type": "object"
+        }
+      },
+      "required": [
+        "tasks"
+      ],
+      "type": "object"
+    },
+    "TaskState": {
```

Comment:
Since TaskState is only used in one place (AnalyzeTextLROResult), I would merge the two.

Reply:
We will be supporting new LRO types starting in a near-future preview version which is now being planned, which will probably have the same structure but different discriminator values.  I think in this case it might make sense to keep it so we can share task state with the other LRO APIs we plan to implement, but let me know if you feel really strongly about this.